### PR TITLE
Mobile market filters

### PR DESCRIPTION
**Pull request by TvrtkoM**

### CODE REVIEW

**Comment by yornaath** [comment]
The arrow here should maybe be aligned vertically with the text?
Screenshot 2023-05-03 at 10 12 46

File: components/markets/market-filter/MarketFiltersSort.tsx [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    return {
+      ...provided,
+      backgroundColor: "white",
+      color: "black",
+      zIndex: 100,
+    };
+  },
+};
+
+const IndicatorSeparator = () => {
```
Comment (Robiquet) [inline components/markets/market-filter/MarketFiltersSort.tsx:60]: Does this do anything?

File: components/markets/market-filter/MarketFiltersSort.tsx [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+  }) => {
+    return (
+      <ReactSelect
+        value={marketsOrderByOptions.find((opt) => opt.value === ordering)}
+        onChange={(v) => {
+          onOrderingChange(v.value);
+        }}
+        options={marketsOrderByOptions}
+        styles={{ ...sortBySelectStyles, ...(styles ?? {}) }}
+        // menuIsOpen={true}
```
Comment (Robiquet) [inline components/markets/market-filter/MarketFiltersSort.tsx:90]: Can this be removed?

**Review by Robiquet — COMMENTED** [review]
Looks nice, only think I noticed was the drop down arrow isn't clickable. Is that an easy fix?
image

File: components/markets/market-filter/MarketFiltersSort.tsx (hunk 1 above)
Comment (TvrtkoM) [reply components/markets/market-filter/MarketFiltersSort.tsx:60]: Yes, it removes vertical line between select value and indicator arrow that react select is showing by default. It is added in `components` prop of ReactSelect

**Review by Robiquet — APPROVED** [review]

**Pull request merged**